### PR TITLE
CSS Conf EU 2018

### PR DESCRIPTION
**Pull request by josectobar**
#8

### CODE REVIEW

**Comment by josectobar** [comment]
#8

**Comment by EddyVinck** [comment]
Hi @josectobar! Thanks for your contribution! I can merge this in if you fix the merge conflicts and update the unique IDs.

File: client/data/conferences.json [hunk 1]
```diff
@@ -11,6 +11,12 @@
       "start_date": "2019-06-01",
       "end_date": "2019-06-02",
       "locations": ["Arena Berlin. Eichenstraße 4, 12435 Berlin, Germany"]
+    },
+    "2": {
```
Comment (EddyVinck) [inline client/data/conferences.json]: To fix the merge conflicts, merge the original master branch and update your IDs. Please re-order them as well so the numbers are in a logical order.

File: client/data/speakers.json [hunk 2]
```diff
@@ -1,67 +1,226 @@
...
-    "54": { "name": "Tatiana Mac" },
-    "55": { "name": "Aga Naplocha" },
-    "56": { "name": "Eli Schutze" },
-    "57": { "name": "Emily Rose" },
-    "58": { "name": "Jenn Turner" },
-    "59": { "name": "Manjula Dube" },
-    "60": { "name": "Rachel White" },
-    "61": { "name": "Sareh Heidari" },
-    "62": { "name": "Whitney Williams" }
+    "0": {
```
Comment (EddyVinck) [inline client/data/speakers.json]: To fix the merge conflicts, merge the original master branch and update your IDs. Please re-order them as well so the numbers are in a logical order.

Also please turn off the formatting.

File: client/data/talks.json [hunk 3]
```diff
@@ -449,6 +449,105 @@
       "video_url": "https://www.youtube.com/watch?v=MBndZddVQdw",
       "video_upload_date": "2019-06-03",
       "conferences": [1]
+    },
+    "50": {
```
Comment (EddyVinck) [inline client/data/talks.json]: To fix the merge conflicts, merge the original master branch and update your IDs. Please re-order them as well so the numbers are in a logical order.

**Review by EddyVinck — CHANGES_REQUESTED** [review]

**Comment by EddyVinck** [comment]
@josectobar Please let me know if this is something you'd still like to work on

**Comment by josectobar** [comment]
I do, give me a couple of hours.

File: client/data/talks.json [hunk 4]
```diff
@@ -470,11 +470,7 @@
     },
     "52": {
       "speakers": [65],
-      "main_title": "Any Site can be a Progressive Web App",
```
Comment (EddyVinck) [inline client/data/talks.json]:
```
 "main_title": "Any Site can be a Progressive Web App",	
      "alternative_titles": [],	
      "categories": [1, 8],	
      "video_url": "https://www.youtube.com/watch?v=GXv4W94qsQ0",	
      "video_upload_date": "2018-06-11",
```
I think you accidentally deleted this @josectobar

Comment (josectobar) [reply client/data/talks.json]: sorry.

Comment (EddyVinck) [reply client/data/talks.json]: No problem! It happens

**Comment by EddyVinck** [comment]
@josectobar Since I merged some larger commits in first I updated the IDs and such for your PR myself. Thank you for your contribution! Remember to join our [Slack](https://join.slack.com/t/conftalks/shared_invite/enQtNzk1MjA1ODQ2NzM3LTRkMmU0YmRhZDEzYmUxZTEyYjhlMWYzOWIyYTU0NDBkMGFlN2U3MjJmNWE5MjM4ZDM1Yzg0M2QwZWUzODllMzM) and [newsletter](https://emailoctopus.com/lists/b6cc50a1-f282-11e9-be00-06b4694bee2a/forms/subscribe) if you haven't already :slightly_smiling_face: 

I'm going to merge this in now

File: client/data/talk_categories.json [hunk 5]
```diff
@@ -35,6 +35,10 @@
     "8": {
       "name": "Progressive Web App",
       "alternative_spelling": ["pwa"]
+    },
+    "9": {
+      "name": "Design Systems",
```
Comment (dios-david) [inline client/data/talk_categories.json:40]: @EddyVinck do we need both `UX` and `Design Systems`?

**Pull request merged**